### DISCUSSION TITLE
Extra colors for the color picker/palette (chosen by you)

### DISCUSSION THREAD

**Discussion by maghiel** [maghiel/skin.eminence.2#12] [Ideas]
Yesterday I've added a shade of grey to the color picker as requested by @Maaaash32.
Now I want to add some more colors to fill up the rows and cells. 
I'm thinking of either widening the picker, iow adding two more colors, or adding to the bottom row (imho the scrolling is fine) iow adding 7 more colors. 
The question is, which colors would *you* like to be added?

**Comment by Maaaash32** [comment]
I just missed the grey color. And I don't have any ideas of other colors.
